composer.json

```diff
@@ -78,7 +78,7 @@
         "scenarios": {
             "opentracing1": {
                 "require": {
-                    "opentracing/opentracing": "1.0.0-beta6"
+                    "opentracing/opentracing": "^1.0"
```

Comment:
The `"extra"` section in the root `composer.json` can be removed altogether as it is no longer used, while the one in `tests/composer.json` is used.

Reply:
Good catch, did not think about that. Thanks